### DISCUSSION TITLE
SideFlipSaga in MarkDown?

### DISCUSSION THREAD

**Discussion by MrBrain295** [MrBrain295/SideFlipSaga#9] [Show and tell]
About SideFlipSaga
==================

![A rabbit](Images/rabbit.jpeg)

This is Sideflipsaga made by Gabe and now Jacob

We post pretty much **what ever** we want **when ever** we want

Our GitHub

Things you can download from us

-   Minecraft worlds (bedrock and java)
-   open source python and java files

[Made partly with Bluefish HTML editor.](http://bluefish.openoffice.nl/)

###### WARNING! this website is made for Firefox some things may not work in Chrome, Safari, etc

**Comment by CasualSideo** [comment]
hmmmmm. I like this idea.

**Reply by MrBrain295** [reply]
thanks @SideFli

**Comment by MrBrain295** [comment]
I moved this to its own repo [here](https://github.com/Sideflipsa-ga/SideFlipSaga-markdown)

**Comment by MrBrain295** [comment]
The top post will become out dated please go to the separate repository.
